packages/rancher-logging/4.10/generated-changes/patch/templates/_helpers.tpl.patch

```diff
@@ -120,6 +120,31 @@
 +{{- end -}}
 +{{- end -}}
 +
++{{/* Implements logic to add kubeAudit loggingOverlay fields to custom loggings based on the cluster type */}}
++{{- define "logging-operator.logging.kube-audit.logging-overlay" -}}
++{{ .Values.additionalLoggingSources.kubeAudit.loggingOverlay }}
++{{ end }}
++
++{{/* Implements logic to add root loggingOverlay fields to custom loggings based on the cluster type */}}
++{{- define "logging-operator.logging.root.logging-overlay" -}}
++{{ .Values.additionalLoggingSources.root.loggingOverlay }}
```

Comment:
I think we need to be careful here to make sure that the existing values field will still work. We don't want to cause an unintended breaking change. So in theory: `.Values.loggingOveraly.spec.fluentd.logLevel`, should be the same as `.Values.fluentd.logLevel` and also the same as `.Values.additionalLoggingSources.root.loggingOverlay.spec.fluentd.logLevel` (based on this line). This is just a single field example across all fields that can potentially configure a value for it.

So as long as they all work (and in expected order) then we should be OK. But currently I think the original overlay value might be the missing bit.